### PR TITLE
feat: support mysql in metadata service

### PR DESCRIPTION
**Pull request by crazy-2020**
### Summary of Changes

Implementation of [RFC-mysql_support_in_metadata_service](https://github.com/amundsen-io/rfcs/blob/master/rfcs/000-mysql-support-in-metadata-service.md)
* Added mysql_proxy 
   * A new config block for mysql use
   * A new proxy: mysql_proxy.py
* Added cli support for rds schema migration 
   * A new cli script: rds_command.py
   * Cli registration in init.py of metadata service
### Tests

- Unit tests for mysql_proxy
- Unit tests for rds_commands

### Documentation

A doc added for how the mysql_proxy works basically and the introduction to schema migration cli.

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes.
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

File: metadata/docs/proxy/mysql.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# MySQL Proxy
+
+MySQL proxy works with [SQLAlchemy ORM](https://docs.sqlalchemy.org/en/13/orm/) and [Amundsen RDS](https://github.com/amundsen-io/amundsenrds)
+containing all the ORM models used for Amundsen databuilder and metadataservice.
+
+# Requirements
+MySQL >= 5.7
```
Comment (feng-tao) [inline metadata/docs/proxy/mysql.md:7]: so it support mysql 8 as well?

File: metadata/metadata_service/client/rds_client.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        """
+        LOGGER.info('Validating schema version.')
+
+        config = self._get_alembic_config()
+        script_directory = script.ScriptDirectory.from_config(config)
+
+        with self.engine.begin() as conn:
+            context = migration.MigrationContext.configure(conn)
+            current_version = context.get_current_revision()
+            current_head = script_directory.get_current_head()
```
Comment (feng-tao) [inline metadata/metadata_service/client/rds_client.py:76]: TIL :)

**Review by feng-tao — APPROVED** [review]
lgtm

File: metadata/docs/proxy/mysql.md (hunk 1 above)
Comment (crazy-2020) [reply metadata/docs/proxy/mysql.md:7]: yes, it supports mysql 8 as well

**Pull request merged**